openmls/tests/test_external_commit.rs

```diff
@@ -120,3 +120,157 @@ fn test_external_commit(ciphersuite: Ciphersuite, backend: &impl OpenMlsCryptoPr
         assert_eq!(got_error, ExternalCommitError::InvalidGroupInfoSignature);
     }
 }
+
+#[apply(ciphersuites_and_backends)]
+fn test_group_info(ciphersuite: Ciphersuite, backend: &impl OpenMlsCryptoProvider) {
+    // Alice creates a new group ...
+    let mut alice_group = {
```

Comment:
While you're at it, do you want to pull this part out into a function? Seems like it's the same setup for all tests in this file.